fe/fe-core/src/main/java/org/apache/doris/qe/SetExecutor.java

```diff
@@ -59,9 +59,24 @@ private void setVariable(SetVar var) throws DdlException {
         }
     }
 
+    private boolean isSessionVar(SetVar var) {
+        return !(var instanceof SetPassVar
```

Comment:
This is error-prone, if other people at a new subclass of `SetVar`, this method may be missed.
How about add a `SetVarType` or something to specify the stmt's type directly?
And write like `return var.setType() == Type.SET_SESSION_VAR`;

Reply:
> This is error-prone, if other people at a new subclass of `SetVar`, this method may be missed. How about add a `SetVarType` or something to specify the stmt's type directly? And write like `return var.setType() == Type.SET_SESSION_VAR`;

Yes, I will modify it.